### DISCUSSION TITLE
Run bot command once in an hour

### DISCUSSION THREAD

**Discussion by mzfr** [python-telegram-bot/python-telegram-bot#2532] [Q&A]
Hey, I want a single user to run the command only once an hour. If they run it again within the hour bot can either respond with some other message or not respond at all.

Is it possible to do this? If yes how?

Thanks

**Comment by Poolitzer** [comment]
Hey. I'd use a [job queue](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions-%E2%80%93-JobQueue) for that. Then you can either check for the existence of this job and based on that tell the user to wait, or you let the job change a variable which you base the wait message on.

**Reply by mzfr** [reply]
Ok thanks, let me try this. Thanks for the reply :)
